packages/conf/src/migrations/normalizeRuntimeConfigModule.test.ts

```diff
@@ -1,62 +1,77 @@
 import { normalizeRuntimeConfigModule } from "./normalizeRuntimeConfigModule"
 
 describe("normalizeRuntimeConfigModule", () => {
-  it("only i18n specified", () => {
-    const actual = normalizeRuntimeConfigModule({
-      runtimeConfigModule: ["../my-i18n", "myI18n"],
-    })
+  test("all defaults", () => {
+    const actual = normalizeRuntimeConfigModule({ runtimeConfigModule: {} })
 
-    expect(actual.runtimeConfigModule).toStrictEqual({
-      TransImportModule: "@lingui/react",
-      TransImportName: "Trans",
-      i18nImportModule: "../my-i18n",
-      i18nImportName: "myI18n",
-    })
+    expect(actual.runtimeConfigModule).toMatchInlineSnapshot(`
```

Comment:
In this whole file, I'd prefer that we revert to using `toStrictEqual` instead of `toMatchInlineSnapshot` because it's both more readable and more "writable".

snapshots are strings. In your IDE, they will display as a bunch of green lines and they are hard to read. 

And have you ever tried writing a snapshot? Again, because it's just a string, you get no help from the language grammar, syntax highlights...

It's not a big deal so feel free to ignore but I think the right use cases for snapshots are elsewhere.

Reply:
I'm a big fan of snapshots. You don't need to touch it manually ever. I changed it to snapshots because it is much easier to maintain, when i added couple new lines into expected output i don't need to go and change every test manually, i just run it with `-u`. That saves a lot of time.

Reply:
okay :). glad to have my opinion challenged 👍